pkgs/development/python-modules/pyslurm/default.nix

```diff
@@ -6,12 +6,10 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     repo = "pyslurm";
     owner = "PySlurm";
-    rev = "69e4f4fd66003b98ddb7da25613fe641d4ae160d";
-    sha256 = "051kafkndbniklxyf0drb360aiblnqcf9rqjbvmqh66zrfya1m28";
+    rev = "a2acbc820da419e308c5817998d2abe78a7b75e6";
```

Comment:
There was a version bump in the upstream repo just yesterday (https://github.com/PySlurm/pyslurm/commit/f717492f8e91599ff413a6789872f43c60fe8d6d). Can't you use that?

Also adding the version to the `name` attribute would probably be good.

Reply:
@andir No I can’t. The version I chose matching slurm 17.02. The latest pyslurm will require slurm 17.11

Also note that this targets stable.

Reply:
ok, makes sense.